client/src/modules/stock/exit/exit.js

```diff
@@ -556,7 +556,7 @@ function StockExitController(
       inventory_uuid : row.inventory.inventory_uuid, // needed for tracking consumption
       uuid : row.lot.uuid,
       quantity : row.quantity,
-      unit_cost : row.lot.unit_cost,
+      unit_cost : row.inventory.wac,
```

Comment:
I wonder if we should the WAC and the stated unit cost in case it is needed for audit and diagnostic purposes?